### DISCUSSION TITLE
Sakai Prime React Tempate (https://sakai.primereact.org/)

### DISCUSSION THREAD

**Discussion by kmahera** [primefaces/community#1704] [PrimeReact Templates] [answered]
Hi I have downloaded latest version of prime react sakai latest version. I want to use this template without next js. How can i do that? Do we have any document created for that. If this will get success I will going to by Freya template.

**Comment by melloware** [comment, marked as answer]
If you look through the Git History tou can see Sakai used to be Create React App without Next.js so you might just grab one of those and start with that.
